site/content/docs/5.2/components/dropdowns.md

```diff
@@ -896,6 +896,12 @@ As part of Boosted's evolving CSS variables approach, dropdowns now use local CS
 
```

Comment:
Answering the question in the description: If we change the class of the dark close button, it might be `.btn-close-inverted` to avoid people to change several times ? Or we keep consistency we other classes and change twice ?